tests/unit/schemas/test_user_strains_schema.py

```diff
@@ -7,7 +7,7 @@
 # Test schema aginast invalid data
 data_no_strain_ids = {"version": "1.0"}
 data_empty_strain_ids = {"strain_ids": [], "version": "1.0"}
-data_invalid_strain_ids = {"strain_ids": [1, 2, 3], "version": "1.0"}
+data_invalid_strain_ids = {"strain_ids": [1, ], "version": "1.0"}
```

Comment:
Why 2 and 3 have been removed? I guess because testing on 1 is enough, but to be sure

Reply:
Yes, only `1` is enough. And the new version of jsonschema changes the order of a list, making the test with multiple items challenging.